pinot-core/src/test/java/org/apache/pinot/queries/TestTextSearchQueries.java

```diff
@@ -89,7 +91,7 @@
   private RecordReader _recordReader;
   Schema _schema;
 
-  private List<IndexSegment> _indexSegments = new ArrayList<>(1);
+  private static List<IndexSegment> _indexSegments = new ArrayList<>(1);
```

Comment:
Revert the change in this file?
Also suggesting rename the test to TextSearchQueriesTest for naming convention.

Reply:
oops. Sorry, this was due to local debugging. Will remove and also reformat